skyreader/plot/plot.py

```diff
@@ -816,35 +814,18 @@ def bounding_box(ra, dec, theta, phi):
                     color=cont_col,
                     linestyle=cont_sty
                 )
-
         plt.legend(fontsize=6, loc="lower left")
-
         # Dump the whole contour
         path = unique_id + ".contour.pkl"
         print("Saving contour to", path)
         with open(path, "wb") as f:
             pickle.dump(saving_contours, f)
 
-        # logic for multiordermaps -> for future developements
-
-        # save multiorder version of the map
-        # multiorder_map = mhealpy.HealpixMap(
-        #     grid_value, uniq_array
-        # )
-        # multiorder_map.write_map(
-        #     f"{unique_id}.skymap_nside_{mmap_nside}.multiorder.fits.gz",
-        #     column_names=["PROBABILITY"],
-        #     extra_header=fits_header,
-        #     overwrite=True,
-        # )
-
         if llh_map:
             column_names = ['2DLLH']
         else:
             # avoid excessively heavy data format for the flattened map
-            equatorial_map[
-                equatorial_map < 1e-16
-            ] = np.mean(
+            equatorial_map[equatorial_map < 1e-16] = np.mean(
                 equatorial_map[equatorial_map < 1e-16]
             )
             column_names = ["PROBABILITY"]
```

Comment:
I think the column names should probably be set by map type here (LLH or probability), and in the case of probability actually "probability density" (I think LVK calls this "PROBDENSITY")

Reply:
I think I already implemented it to do exactly what you suggested. I called the probability density column "PROBABILITY DENSITY [deg-2]". I included the unit so it is clear that it is degrees and not radiants.